### DISCUSSION TITLE
Hefei-NAMD

### DISCUSSION THREAD

**Discussion by yht-0816** [deepmodeling/abacus-develop#5279] [General]
### Details

Hello everyone, may I ask you how to Hefei ABACUS+Hefei-namd (tutorial: https://mcresearch.github.io/abacus-user-guide/abacus-namd.html),  the output files of ABACUS can be converted into NATXT and EIGTXT files needed by Hefei NAMD by running NAC.py (this step is very fast when vasp was used before), there is no error and output in 3 hours.The following is  script related to NAC.py.

### Have you read FAQ on the online manual http://abacus.deepmodeling.com/en/latest/community/faq.html

- [X] Yes, I have read the FAQ part on online manual.

### Task list for Issue attackers (only for developers)

- [ ] Understand the problem or question described by the user.
- [ ] Check if the issue is a known problem or has been addressed in the documentation.
- [ ] Test the issue or problem on a similar system or environment, if possible.
- [ ] Identify the root cause or provide clarification on the user's question.
- [ ] Provide a step-by-step guide, including any necessary resources, to resolve the issue or answer the question.
- [ ] If the issue is related to documentation, update the documentation to prevent future confusion (optional).
- [ ] If the issue is related to code, consider implementing a fix or improvement (optional).
- [ ] Review and incorporate any relevant feedback from users or developers.
- [ ] Ensure the user's issue is resolved or their question is answered and close the ticket.

**Comment by WHUweiqingzhou** [comment]
@dyzheng could you have a look?

**Comment by AsTonyshment** [comment]
I went to Chizhou, Anhui last month and learnt a bit of Hefei-NAMD, so I will try to look into it in the near future.
